tests/e2e/actions.go

```diff
@@ -250,6 +258,121 @@ func (tr Chain) submitTextProposal(
 	tr.waitBlocks(action.Chain, 1, 10*time.Second)
 }
 
+type UpdateConsumerChainAction struct {
+	Chain               ChainID
+	From                ValidatorID
+	ConsumerChain       ChainID
+	NewOwner            string
+	SpawnTime           uint
+	InitialHeight       clienttypes.Height
```

Comment:
Reorder `SpawnTime` and `InitialHeight` to respect `ConsumerInitializationParameters` order.